src/custom/components/Header/index.tsx

```diff
@@ -1,8 +1,16 @@
-import HeaderMod from './HeaderMod'
+import HeaderMod, { UniIcon, HeaderFrame } from './HeaderMod'
 import styled from 'styled-components'
 
 export const Header = styled(HeaderMod)`
   // CSS Override
+
+  ${UniIcon} {
+    display: flex;
+  }
+
+  ${HeaderFrame} {
+    border-bottom: 1px solid #3A3B5A;
```

Comment:
@anxolin For some reason, this one doesn't get applied. Should I export HeaderFrame separately just like with Header?

e.g. `export const Header = styled(HeaderMod)`

Reply:
let me review it

Reply:
Here u have: 884d6e8a7b61cbafe69196b95907d4b13c61828c :)